packages/stats/gbstats/bayesian/tests.py

```diff
@@ -2,6 +2,7 @@
 from dataclasses import field
 from typing import List, Optional, Tuple, Union
 
+import scipy.stats
 import numpy as np
 from pydantic.dataclasses import dataclass
 from scipy.stats import norm  # type: ignore
```

Comment:
Let's remove this import and just call `scipy.stats.norm` where I use `norm` below to be consistent with imports.

Reply:
done

Reply:
I think then we can remove this import that isn't used anymore.